<?xml version="1.0"?>
<launch>
    <!-- Startup UR5 and Kinect -->
    <include file="$(find tams_ur5_setup_bringup)/launch/tams_ur5_setup_floor_kinect.launch"/>
    <param name="move_group/constraint_approximations_path" value="$(find project16_manipulation)/cadb_20000_bagger"/>

    <!-- Startup image processing -->
    <include file="$(find project16_image_processing)/launch/regular.launch"/>

    <!-- Startup manipulation -->
    <include file="$(find project16_manipulation)/launch/manipulation.launch"/>
    
    <!-- Startup coordinator -->
    <include file="$(find project16_coordinator)/launch/coordinator.launch"/>
    
    <!-- Startup GUI -->
    <include file="$(find project16_gui)/launch/rosbridge.launch"/>
    
    <!-- Startup Rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find project16_coordinator)/rviz/startup.rviz" />
</launch>
